### PR TITLE
Editorial things

### LINKED ISSUE #113: Merge field parameters into one table

### LINKED ISSUE #168: `format_custom()` function signature is too long for
image

### PR DESCRIPTION
**Pull request by cjpatton**
Closes #113.
Closes #168.

### CODE REVIEW

File: draft-irtf-cfrg-vdaf.md [hunk 1]
```diff
@@ -1219,7 +1219,7 @@ discrete Fourier transform, as this allows for fast polynomial interpolation.
 (One example is Prio3 ({{prio3}}) when instantiated with the generic FLP of
 {{flp-generic-construction}}.) Specifically, a field is said to be
 "FFT-friendly" if, in addition to satisfying the interface described in
-{{field}}, it implements the following method:
+{{fields}}, it implements the following method:
```
Comment (divergentdave) [inline draft-irtf-cfrg-vdaf.md:1222]: I think we still want to point to the section above, not the field parameters table.

File: draft-irtf-cfrg-vdaf.md [hunk 2]
```diff
@@ -1452,7 +1440,7 @@ The parameters provided by a concrete FLP are listed in {{flp-param}}.
 | `VERIFIER_LEN`   | Length of the verifier message generated by querying the input and proof |
 | `Measurement`    | Type of the measurement   |
 | `AggResult`      | Type of the aggregate result |
-| `Field`          | As defined in ({{field}}) |
+| `Field`          | As defined in ({{fields}}) |
```
Comment (divergentdave) [inline draft-irtf-cfrg-vdaf.md:1443]: This should point to `#field` as well, as that's where the class is defined.

File: draft-irtf-cfrg-vdaf.md [hunk 3]
```diff
@@ -2656,7 +2644,7 @@ state across evaluations. See {{idpf-poplar}} for details.
 | BITS       | Length in bits of each input string |
 | VALUE_LEN  | Number of field elements of each output value |
 | KEY_SIZE   | Size in bytes of each IDPF key |
-| FieldInner | Implementation of `Field` ({{field}}) used for values of inner nodes |
+| FieldInner | Implementation of `Field` ({{fields}}) used for values of inner nodes |
```
Comment (divergentdave) [inline draft-irtf-cfrg-vdaf.md:2647]: Ditto

File: draft-irtf-cfrg-vdaf.md (hunk 2 above)
Comment (cjpatton) [reply draft-irtf-cfrg-vdaf.md:1443]: Whoopsies, this is a regex whoopsy daisy! Thanks

**Review by divergentdave — APPROVED** [review]

**Pull request merged**